build/rewrite_test.go

```diff
@@ -95,3 +96,43 @@ func TestRewriterWithRewriter(t *testing.T) {
 		t.Error("Original Printer should not equal Modified Printer")
 	}
 }
+
+func TestBUILDFileOrderingRewriter(t *testing.T) {
```

Comment:
What's the reason this test is implemented separately, not as a test file under `build/testdata`? Rewrites are also done after ordinary formatting, aren't they?